packages/meditrak-server/src/apiV2/verifyEmail.js

```diff
@@ -78,7 +54,7 @@ export const requestResendEmail = async (req, res) => {
     throw new FormValidationError(`Unable to send verification email to ${emailAddress}`);
   }
 
-  sendEmailVerification(user);
+  await sendEmailVerification(user);
```

Comment:
I'd say this was originally intentional, as we don't need to wait for the email to be sent before we respond, but it's probably a good call in case there's an error in sending it.